client/dashboard/profile-wizard/steps/industry.js

```diff
@@ -130,6 +159,22 @@ class Industry extends Component {
 	render() {
 		const { industries } = onboarding;
 		const { error, selected, textInputListContent } = this.state;
+		const { locationSettings } = this.props;
+		const region = getCurrencyRegion(
+			locationSettings.woocommerce_default_country
+		);
+
+		const filteredIndustries = Object.keys( industries ).filter(
```

Comment:
Nitpick: industries only need to be filtered if the region isn't the US.

Reply:
👍